OPHD/States/MapViewStateIO.cpp

```diff
@@ -135,6 +136,19 @@ void MapViewState::save(const std::string& filePath)
 }
 
 
+void MapViewState::serializeProperties(NAS2D::Xml::XmlElement* element, const Planet::Attributes& planetAttributes)
+{
+	XmlElement* properties = new XmlElement("properties");
+	element->linkEndChild(properties);
```

Comment:
External concern to this PR, though I wanted to note it may be better to return the new `XmlElement` from the function, and perform the `linkEndChild` at the point of call, rather than passing in an `XmlElement` that is never used for any other purpose.

That may also play better with some of the XML support code in NAS2D, such as `dictionaryToAttributes`, which generates and returns an `XmlElement`, with a `Dictionary` of key/value pairs added as XML attributes.